hazelcast/src/main/java/com/hazelcast/internal/metrics/MetricDescriptorConstants.java

```diff
@@ -606,6 +610,8 @@ public final class MetricDescriptorConstants {
     public static final String TSTORE_HLOG_PAGING_FREQUENCY_AVG = "tstore.hlog.paging.frequency.avg";
     public static final String TSTORE_HLOG_PAGING_FREQUENCY_MIN = "tstore.hlog.paging.frequency.min";
     public static final String TSTORE_HLOG_PAGING_FREQUENCY_MAX = "tstore.hlog.paging.frequency.max";
+    public static final String TSTORE_DISK_USAGE = "tstore.disk.usage";
```

Comment:
Maybe this should be device as well.

Reply:
See https://github.com/hazelcast/hazelcast-enterprise/pull/5178/commits/381850ae10e096d168f02108636351c5e8511a9f.

This metrics is now both aggregated per map and per member. If I use device here, I think it will create a confusion. Because one member can have multiple devices. WDYT?

Reply:
I mean, the device might be not a disk (later, network share, S3, etc). We use the term device for distinguishing the parts in the memory and not in the memory.

Reply:
Ok, after standup I will handle this, you are right. We have `DiskTierConfig` though.

Reply:
Besides renaming, I also added per device aggregated version of `device.usage`.

https://github.com/hazelcast/hazelcast/pull/21911/commits/d4b65bcc1f4eba7d5e7ac6392e2735b05e5707e9
https://github.com/hazelcast/hazelcast-enterprise/pull/5178/commits/f1b71ced64cbe9ca66c8c570f557a6ff6e8e07f5